app/src/main/java/com/lagradost/cloudstream3/ui/settings/extensions/PluginAdapter.kt

```diff
@@ -101,15 +101,6 @@ class PluginAdapter(
             return findClosestBase2(target, current * 2, max)
         }
 
-        @Test
-        fun testFindClosestBase2() {
```

Comment:
Why do you remove the tests?

Reply:
Tests shouldn't be included in production code?

Reply:
I does not matter, you should never remove tests.

Reply:
Can I move it to the test package? Or does it have to stay there?

Reply:
It does not matter where it it placed, but don't remove it.